docs-md/tutorials/basics-docker.md

```diff
@@ -85,9 +85,9 @@ docker run --network tutorials_default --rm --name datagen-users \
     ksql-datagen \
         bootstrap-server=kafka:39092 \
         quickstart=users \
-        format=json \
+        format=avro \
```

Comment:
I've switched this to Avro as I've had to switch later examples from Avro to Json as they have non-STRING keys and Avro schema inference currently only works with STRING keys :(